### ISSUE TITLE
Bug: Freetext Import not opening (v2.5.0)

### ISSUE THREAD

**Issue by samitainio** [MISP/MISP#9951]
### Actual behavior

I cannot open up the Freetext Import menu via either pressing F or through Populate from... --> Freetext Import

### Expected behavior

I can open up the Freetext Import menu via either pressing F or through Populate from... --> Freetext Import

### Steps to reproduce

Either press F or try adding via the Populate from... --> Freetext Import. 

### Version

v2.5.0 (04e5d39148eb1008bfa64fc389f14ab611e655c3) 

### Operating System

Ubuntu

### Operating System version

24.04.1

### PHP version

8.3.6

### Relevant log output

```shell
==> /var/www/MISP/app/tmp/logs/error.log <==

2024-10-05 20:15:37 Error: [TypeError] Attribute::__construct(): Argument #1 ($flags) must be of type int, array given
Request URL: /events/freeTextImport/4730?_=1728159333199
Stack Trace:
#0 [internal function]: Attribute->__construct()
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Utility/ClassRegistry.php(169): ReflectionClass->newInstance()
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/Helper/FormHelper.php(171): ClassRegistry::init()
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/Helper/FormHelper.php(204): FormHelper->_getModel()
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/Helper/FormHelper.php(354): FormHelper->_introspectModel()
#5 /var/www/MISP/app/View/Events/free_text_import.ctp(2): FormHelper->create()
#6 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/View.php(971): include('...')
#7 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/View.php(933): View->_evaluate()
#8 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/View.php(473): View->_render()
#9 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(973): View->render()
#10 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(200): Controller->render()
#11 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke()
#12 /var/www/MISP/app/webroot/index.php(99): Dispatcher->dispatch()
#13 {main}

2024-10-05 20:19:49 Error: [TypeError] Attribute::__construct(): Argument #1 ($flags) must be of type int, array given
Request URL: /events/freeTextImport/4730?_=1728159333201
Stack Trace:
#0 [internal function]: Attribute->__construct()
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Utility/ClassRegistry.php(169): ReflectionClass->newInstance()
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/Helper/FormHelper.php(171): ClassRegistry::init()
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/Helper/FormHelper.php(204): FormHelper->_getModel()
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/Helper/FormHelper.php(354): FormHelper->_introspectModel()
#5 /var/www/MISP/app/View/Events/free_text_import.ctp(2): FormHelper->create()
#6 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/View.php(971): include('...')
#7 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/View.php(933): View->_evaluate()
#8 /var/www/MISP/app/Lib/cakephp/lib/Cake/View/View.php(473): View->_render()
#9 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(973): View->render()
#10 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(200): Controller->render()
#11 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke()
#12 /var/www/MISP/app/webroot/index.php(99): Dispatcher->dispatch()
#13 {main}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tomking2** [comment]
Should be fixed by 1399a8d2f00c3f2ec628716b8836566d41888735

**Comment by samitainio** [comment]
Confirming fix!
